Backpack/Button/Classes/BPKButton.m

```diff
@@ -130,16 +132,26 @@ - (BOOL)hasTitle {
     return self.titleLabel.text.length > 0 && self.currentAttributedTitle != nil;
 }
 
+- (BOOL)hasImage {
```

Comment:
I think it would be cleaner to have a property `hasIcon` which returns whether the icon property is set.
In my mind, `hasImage` would always be true when `loading` is true

Reply:
`hasImage` is independant of `loading`. For example a text and icon button still has an image if it's loading. However I'm open to calling this `hasIcon`

Reply:
Yeah maybe `hasIcon` would clear things up 👍